lib/models/types/moment.js

```diff
@@ -23,7 +23,7 @@ SchemaTypeMoment.prototype.cast = function(value, data) {
   const { options } = this;
   value = toMoment(value);
 
-  if (options.language) value = value.locale(options.language);
+  if (options.language) value = value.locale(toMomentLocale(options.language));
```

Comment:
`options.language` always is `undefined`.
This is a bug, or this is unnecessary code.

Reply:
As far as I know, `models/types/moment` is only used in `Post` and `Page` model.

What happened then is still a mystery. Only tommy351 knows.